bin/shabka

```diff
@@ -0,0 +1,383 @@
+#! /usr/bin/env bash
+
+set -euo pipefail
+
+declare __VERBOSE="${__VERBOSE:-0}"
+
+echoErr() { >&2 echo -e "\e[1;31m[ERR] ${*}\e[0m" ; }
+echoWarn() { >&2 echo -e "\e[1;33m[WARN] ${*}\e[0m" ; }
+echoInfo() { >&2 echo -e "\e[1;34m[INFO] ${*}\e[0m" ; }
+echoDebug1() {
+  if [[ "${__VERBOSE}" -lt 1 ]]; then
+    return
+  fi
+  >&2 echo -e "\e[1;35m[DEBUG1] ${*}\e[0m"
+}
+echoDebug2() {
+  if [[ "${__VERBOSE}" -lt 2 ]]; then
+    return
+  fi
+  >&2 echo -e "\e[1;35m[DEBUG2] ${*}\e[0m"
+}
+echoDebug3() {
+  if [[ "${__VERBOSE}" -lt 3 ]]; then
+    return
+  fi
+  >&2 echo -e "\e[1;35m[DEBUG3] ${*}\e[0m"
+}
+echoSuccess() { >&2 echo -e "\e[1;32m[SUCCESS] ${*}\e[0m" ; }
```

Comment:
I suggest combining these and https://github.com/kalbasit/shabka/blob/d877e655176c64f2a015a18b8a50543b9c2dd9ae/os-specific/darwin/setup.sh#L12-L18 into some sort of helper.
 
FYI, all colors are at https://github.com/kalbasit/shabka/blob/d877e655176c64f2a015a18b8a50543b9c2dd9ae/modules/home/software/zsh/init-extra.zsh#L275-L311

Reply:
I don't think I can rely in whatever is defined in shabka, as we don't know where is shabka at the start of the script.

Reply:
Good point.